src/main/resources/interface-specification.yml

```
@@ -1574,6 +1574,37 @@ paths:
               schema:
                 type: integer
               description: Time interval in milliseconds. Allowed requests will be constantly replenished during the interval. At the end of the interval the max allowed requests will be available
+  /eservices:
+    parameters:
+      - $ref: '#/components/parameters/CorrelationIdHeader'
+      - $ref: '#/components/parameters/IpAddress'
+    post:
+      security:
+        - bearerAuth: [ ]
+      tags:
+        - catalog
+      summary: Create a new e-service
```

Let's use the same naming everywhere
```suggestion
      summary: Create a new EService
```